### ISSUE TITLE
Finetuning with 1% and 10%

### ISSUE THREAD

**Issue by zlapp** [facebookresearch/swav#13]
Hi,

Thanks for sharing this awesome repo.

I was wondering in the eval script how is the finetuning on 1% and 10% of imagenet done?

Here it looks like the entire folder is used for the dataset:
https://github.com/facebookresearch/swav/blob/139623b23be7d1f0caeea7aa5fb292f811df22d7/eval_linear.py#L103

Was 1% and 10% of train imagenet preselected and placed into separate folders?

**Comment by mathildecaron31** [comment]
Hi, I have not included the code for fine-tuning on 1% and 10% of labels in this public repo as it is quite similar to regular ImageNet training. I can add the code for finetuning if that helps you in the following days.

**Comment by mathildecaron31** [comment]
`eval_linear.py` is for linear probes only (not fine-tuning)

**Comment by mathildecaron31** [comment]
https://github.com/facebookresearch/swav/commit/57eac591b2a26eb964022342533e80d0677b825d
